testdata/p4_16_samples/pna-example-tcp-connection-tracking-err-1.p4

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 */
 
 #include <core.p4>
-#include "../pna.p4"
+#include "pna.p4"
```

Comment:
This doesn't require change in path? dpdk/pna.p4?

Reply:
@usha1830 , I think this is common test case and may not require dpdk/pna.p4.

Reply:
Ok . Got it . Thanks!